src/app/[lang]/app/search-pro/page.tsx

```diff
@@ -0,0 +1,31 @@
+'use client';
+import { ProSearchField } from '@/modules/core/containers/pro-search-field';
+import styles from './search-pro.module.scss';
+import { ProfessionalSearchCard } from '@/modules/user/containers/professional-search-card';
+import { trpc } from '@/modules/core/utils/trpc.utils';
+import { Typography } from '@/modules/core/components/typogrpahy';
+
+export default function SearchProPage() {
+  const { data: professionalList } = trpc.professional.list.useQuery({
+    limit: 6,
+  });
+
+  return (
+    <div className={styles.root}>
+      <section className='w-full max-w-[920px] mx-auto flex pt-14 pb-10'>
+        <ProSearchField />
+      </section>
+      <section className={styles.pageContent}>
+        <div className='pageContent py-12 gap-y-8 flex flex-col'>
+          <Typography variant='body1'>Search results</Typography>
+          <div className='grid grid-cols-3 gap-6'>
+            {professionalList?.map((pro) => (
+              // @ts-ignore todo: Will be fixed later. Expected different api query with different response.
+              <ProfessionalSearchCard professional={pro} key={pro.id} />
+            ))}
+          </div>
+        </div>
+      </section>
+    </div>
+  );
+}
```

Comment:
I think we need to add pagination here, backend has property `offset`, which tels how many items we should skip, by using this we should be able paginate through professionals. If needed we can discuss another format for pagination.

Reply:
Yes, but I think we'll need another API method here. We'll need to implement a few filters there. We can discuss the API implementation later. For now, the UI of filters isn't ready.